server/src/routes/router.js

```diff
@@ -19,26 +21,26 @@ export default (app, express) => {
   // API ROUTES
 
   //////////////////////////////////////////////
-  //Handling Team Oauth
+  //Handling Oauth
   //////////////////////////////////////////////
-  app.get('/slack/teams/auth', teamController.addTeam);
+  app.get('/slack/teams/auth', authTeamController.authTeam);
+  app.get('/slack/users/auth', authUserController.checkAuthCode);
+  app.get('/slack/users/data', requireAuth, authUserController.getUserData);
 
   //////////////////////////////////////////////
-  //Handling Users
+  //Handling Team
   //////////////////////////////////////////////
-
-  // TODO: fix this so that it is in the userController!
-  app.get('/slack/users/:slackUserId', userController.findUser);
-  app.post('/slack/users', userController.addUsers);
-  app.put('/slack/users', userController.updateLocation)
-
-  // Grabbing user data
-  app.get('/slack/users/data', requireAuth, userController.getUserData);
+  app.get('/api/teams', teamController.findAllTeams);
+  app.get('/api/teams/:slackTeamId', teamController.findTeam);
+  app.post('/api/teams', teamController.addTeam);
 
   //////////////////////////////////////////////
-  //Handling Oauth
+  //Handling Users
   //////////////////////////////////////////////
-  app.get('/slack/users/auth', userController.checkAuthCode);
+  app.get('/api/users/:slackUserId', userController.findUser);
+  app.post('/api/users', userController.addUsers);
```

Comment:
Are we still using this function?  I feel like we should just have 
`app.post('/api/users', userController.addUser);
app.put('/api/users/location', userController.updateUser);
`
The post create a single new user and the put updates a user based off the input options